go/zeusclient/zeusclient.go

```diff
@@ -26,41 +26,59 @@ const (
 // man signal | grep 'terminate process' | awk '{print $2}' | xargs -I '{}' echo -n "syscall.{}, "
 var terminatingSignals = []os.Signal{syscall.SIGHUP, syscall.SIGINT, syscall.SIGKILL, syscall.SIGPIPE, syscall.SIGALRM, syscall.SIGTERM, syscall.SIGXCPU, syscall.SIGXFSZ, syscall.SIGVTALRM, syscall.SIGPROF, syscall.SIGUSR1, syscall.SIGUSR2}
 
-func Run(args []string, input io.Reader, output *os.File) int {
+func Run(args []string, input io.Reader, output *os.File, stderr *os.File) int {
 	if os.Getenv("RAILS_ENV") != "" {
 		println("Warning: Specifying a Rails environment via RAILS_ENV has no effect for commands run with zeus.")
 		println("As a safety precaution to protect you from nuking your development database,")
 		println("Zeus will now cowardly refuse to proceed. Please unset RAILS_ENV and try again.")
 		return 1
 	}
 
+	// if stdout is a terminal, assume that stderr is a terminal as well
```

Comment:
Not sure if this is a save assumption, probably better to check either and do the right thing.